src/Esprima/Esprima.csproj

```diff
@@ -43,16 +43,15 @@
 
   <!-- For checking generated output-->
 
-  <!--
   <PropertyGroup>
```

Comment:
are these changes really needed?

Reply:
I'd got a bunch of annoying CS0121 warnings in the latest VS 2022 before I made this change. It seemed to fix the issue but I'm not 100% sure. However, it doesn't do any harm if it didn't.